app/upgrades.go

```diff
@@ -9,9 +9,32 @@ import (
 	v2 "github.com/Stride-Labs/stride/v4/app/upgrades/v2"
 	v3 "github.com/Stride-Labs/stride/v4/app/upgrades/v3"
 	v4 "github.com/Stride-Labs/stride/v4/app/upgrades/v4"
+	v5 "github.com/Stride-Labs/stride/v4/app/upgrades/v5"
 	claimtypes "github.com/Stride-Labs/stride/v4/x/claim/types"
+
+	authzkeeper "github.com/cosmos/cosmos-sdk/x/authz/keeper"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// AuthzHeightAdjustmentUpgradeStoreLoader is used to delete the authz store with the
+// wrong height and then re-add authz store with the right height
+func AuthzHeightAdjustmentUpgradeStoreLoader(upgradeHeight int64) baseapp.StoreLoader {
```

Comment:
now that's we're not adding authz back in the same upgrade, I don't think this function is necessary. 
I think just doing the following in setupUpgradeHandlers is sufficient. 
```go
case "v5":
    storeUpgrades = &storetypes.StoreUpgrades{
	Deleted: []string{claimtypes.StoreKey},
    }
}
```